src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newBot = {};
+
+  for (const keys in robot) {
+    newBot[robot[keys]] = keys;
+  }
```

Comment:
Everything is great, but only one little comment which is really super unimportant but still. )
You better name const keys in singular (just key) because it contains only one value.